src/types/block.ts

```diff
@@ -187,13 +186,13 @@ export interface BlockTransaction {
   /** The eval deltas for the block */
   dt?: BlockTransactionEvalDelta
   /** Asset ID when an asset is created by the transaction */
-  caid?: number
```

Comment:
From memory we use these types from the message pack decode. Do they definitely get decoded to bigint correctly?

Reply:
thank you for picking this up, indeed the type should be number